mcf-browser/src/back-end/server-api.ts

```diff
@@ -37,9 +37,32 @@ type ParseFileResponse = {
  *     parsing error message objects.
  */
 async function readFileList(fileList: Blob[]) {
-  let curTmcf = null;
+  // Clear previously stored files
+  clearFiles();
+
   const finalReturn: ParseFileResponse = {'errMsgs': [], 'localNodes': []};
 
+  // Find TMCF file, if it exists
+  let tmcfFile = null;
+  for (const file of fileList) {
+    const fileName = (file as File).name;
+    const fileExt = fileName.split('.').pop();
+
+    if(fileExt === "tmcf"){
+      if (tmcfFile) {
+        // If another TMCF file was found, throw an error
+        finalReturn['errMsgs'] = finalReturn['errMsgs'].concat([{
+          'file': fileName,
```

Comment:
maybe instead of the current file that is going to be used, we can put the file that got skipped (tmcfFile)?

Reply:
Good catch! Updated in the next commit